tiles/src/main/java/com/protomaps/basemap/feature/RegionInfos.java

```diff
@@ -7,107 +7,961 @@
+    Sarthe|FR-72|8|11|Q12740
+    Savoie|FR-73|8|11|Q12745
+    Haute-Savoie|FR-74|8|11|Q12751
+    Seine-Maritime|FR-76|8|11|Q12758
+    Seien-et-Marne|FR-77|8|11|Q12753
+    Yvelines|FR-78|8|11|Q12820
+    Deux-Sèvres|FR-79|8|11|Q12765
+    Somme|FR-80|8|11|Q12770
+    Tarn|FR-81|8|11|Q12772
+    Tarn-et-Garonne|FR-82|8|11|Q12779
+    Var|FR-83|8|11|Q12789
+    Vaucluse|FR-84|8|11|Q12792
+    Vendée|FR-85|8|11|Q12798
+    Vienne|FR-86|8|11|Q12804
+    Haute-Vienne|FR-87|8|11|Q12808
+    Vosges|FR-88|8|11|Q3105
+    Yonne|FR-89|8|11|Q12816
+    Territoire de Belfort|FR-90|8|11|Q12782
+    Essonne|FR-91|8|11|Q3368
+    Val-d'Oise|FR-95|8|11|Q12784
+    Guyane française|FR-GF|8|11|Q3769
+    Guadeloupe|FR-GP|8|11|Q17012
+    Martinique|FR-MQ|8|11|Q17054
+    La Réunion|FR-RE|8|11|Q17070
+    Mayotte|FR-YT|8|11|Q17063
     """;
 
-  public record RegionInfo(String name, String countryIsoCode, double minZoom, double maxZoom) {}
+  public record RegionInfo(String name, String regionIsoCode, double minZoom, double maxZoom, String wikidata) {}
 
-  private static final HashMap<String, RegionInfo> regionInfoByName;
-  static RegionInfo unknownInfo = new RegionInfo("UNKNOWN_REGION", "XX", 8.0, 11.0);
+  private static final HashMap<String, RegionInfo> regionInfoByISO;
+  private static final HashMap<String, RegionInfo> regionInfoByWikidata;
+  static RegionInfo unknownInfo = new RegionInfo("UNKNOWN_REGION", "XX", 8.0, 11.0, "QXXX");
 
   static {
-    regionInfoByName = new HashMap<>();
+    regionInfoByISO = new HashMap<>();
     Scanner s = new Scanner(data);
     while (s.hasNextLine()) {
       String line = s.nextLine();
       String[] parts = line.split("\\|");
-      regionInfoByName.put(parts[0],
-        new RegionInfo(parts[0], parts[1], Double.parseDouble(parts[2]) - 1.0, Double.parseDouble(parts[3]) - 1.0));
+      regionInfoByISO.put(parts[1],
+        new RegionInfo(parts[0], parts[1], Double.parseDouble(parts[2]) - 1.0, Double.parseDouble(parts[3]) - 1.0, parts[4]));
     }
   }
 
-  public static RegionInfo getByName(SourceFeature sf) {
-    var name = sf.hasTag("name:en") ? sf.getString("name:en") : sf.getString("name");
-    if (regionInfoByName.containsKey(name)) {
-      return regionInfoByName.get(name);
+  static {
+    regionInfoByWikidata = new HashMap<>();
+    Scanner s = new Scanner(data);
+    while (s.hasNextLine()) {
+      String line = s.nextLine();
+      String[] parts = line.split("\\|");
+      regionInfoByWikidata.put(parts[4],
+        new RegionInfo(parts[0], parts[1], Double.parseDouble(parts[2]) - 1.0, Double.parseDouble(parts[3]) - 1.0, parts[4]));
+    }
+  }
+
+  public static RegionInfos.RegionInfo getByISO(SourceFeature sf) {
+    // ISO codes aren't always included in extracts, oddly, so introspect which tags are
+    //sf.tags().forEach((key, value) -> System.out.printf("%s: %s\n", key, value));
+
+    var isoCode = sf.hasTag("ISO3166-2") ? sf.getString("ISO3166-2") : (
```

Comment:
Even though OSM has this tag, by the time we see the file via GeoFabrik and Planetiler it's gone!?

Reply:
Referring to a entity like https://www.openstreetmap.org/relation/162112 right?